static/docs/install/index.md

```diff
@@ -7,8 +7,17 @@ Please double check that you don't already have DVC (for example running
 - [Install on Windows](/doc/install/windows)
 - [Install on Linux](/doc/install/linux)
 
-You can also [install the development version](/doc/install/pre-release), before
-the official release.
+## Advanced options
 
-It is recommended to setup [shell completion](/doc/install/completion) as well.
-And check out these [plugins](/doc/install/plugins), which might be useful too.
+- Shell completion is included with certain installation methods
+  ([Debian/UbuntuFedora/CentOS Linux repositories](/doc/install/linux#install-from-repository)
+  and
+  [MacOS Homebrew](http://localhost:3000/doc/install/macos#install-with-brew)),
+  and it can be easily [setup manually](/doc/install/completion) for most
+  others.
```

Comment:
@efiop @mroutis can you confirm this is correct? Specifically the "for most others" part. By this I mean that we don't have instructions to setup Bash completion except for MacOs and Debian/Ubuntu. (See https://dvc.org/doc/install/completion.) Or is this doc incomplete? Thanks!

Reply:
`http://localhost:3000/doc/install/macos#install-with-brew` - localhost? )

Also, I would simplify it on the index page - "Shell completion is enabled by default in certain cases, if it does not work please see this instruction". No need to go into details too much

Reply:
Oops. fixed `localhost` URLs.

OK, simplified note, then.